django-backend/fecfiler/transactions/views.py

```diff
@@ -106,45 +106,62 @@ class TransactionListPagination(pagination.PageNumberPagination):
     page_size_query_param = "page_size"
 
 
-class TransactionViewSetBase(CommitteeOwnedViewSet, ReportViewMixin):
-    """ """
-
-    filter_backends = [filters.OrderingFilter]
-    ordering = ["-created"]
-
-
 class TransactionViewSet(CommitteeOwnedViewSet, ReportViewMixin):
     queryset = Transaction.objects.all().alias(
         name=Coalesce(
             Coalesce(
-                "schedule_b__payee_organization_name",
                 "schedule_a__contributor_organization_name",
+                "schedule_b__payee_organization_name",
+                "schedule_c__lender_organization_name",
+                "schedule_d__creditor_organization_name",
+                "schedule_e__payee_organization_name",
             ),
             Concat(
                 Coalesce(
-                    "schedule_b__payee_last_name", "schedule_a__contributor_last_name"
+                    "schedule_a__contributor_last_name",
+                    "schedule_b__payee_last_name",
+                    "schedule_c__lender_last_name",
+                    "schedule_d__creditor_last_name",
+                    "schedule_e__payee_last_name",
                 ),
                 Value(", "),
                 Coalesce(
-                    "schedule_b__payee_first_name", "schedule_a__contributor_first_name"
+                    "schedule_a__contributor_first_name",
+                    "schedule_b__payee_first_name",
+                    "schedule_c__lender_first_name",
+                    "schedule_d__creditor_first_name",
+                    "schedule_e__payee_first_name",
                 ),
                 output_field=TextField(),
             ),
-        )
+        ),
+        sort_amount=Coalesce(
```

Comment:
`amount` and `date` are already working like this in the Transaction manager.  they're just missing schedule e